go/cmd/dolt/commands/schcmds/tags.go

```diff
@@ -87,24 +89,58 @@ func (cmd TagsCmd) Exec(ctx context.Context, commandStr string, args []string, d
 			return 0
 		}
 	}
-	for i := 0; i < len(tables); i++ {
-		tables[i] = fmt.Sprintf("'%s'", tables[i])
+
+	var headerSchema = sql.Schema{
+		{Name: "table", Type: sql.Text, Default: nil},
+		{Name: "column", Type: sql.Text, Default: nil},
+		{Name: "tag", Type: sql.Uint64, Default: nil},
 	}
 
-	//TODO: implement REGEXP_SUBSTR in go-mysql-server and use it here instead of SUBSTR, as this will eventually break
-	queryStr := fmt.Sprintf("SELECT table_name AS 'table', column_name AS 'column', "+
-		"SUBSTR(extra, 5) AS tag FROM information_schema.columns WHERE table_name IN (%s)", strings.Join(tables, ","))
+	rows := make([]sql.Row, 0)
+
+	for _, tableName := range tables {
+		table, ok, err := root.GetTable(ctx, strings.ToLower(tableName))
```

Comment:
This won't work in all cases. root.GetTable is case sensitive. So it's not enough to just lower-case the input like this.

You either need to get all the tables from the root and then look for case-insensitive matches with the tables given on the command line, or you need to implement a case-insensitive version of root.GetTable. The former is probably a smaller change, but think about what you would prefer.

Should be a unit / bats test for this case insensitive matching.

Reply:
Does the GetTableInsensitive in root_val solve this?